### ISSUE TITLE
New ContentType and ContentTypes list admin issues

### ISSUE THREAD

**Issue by jptissot** [OrchardCMS/OrchardCore#4311]
With the latest version deployed on nuget, I can no longer create new ContentItems in the admin. 

The New button is not displayed and the ContentTypes list no longer show stereotypes or create buttons. 

This does work fine when using OC source code directtly. 

The latest version on myget that works is 1.0.0-beta3-72520

**Comment by deanmarcussen** [comment]
The problems looks like it's related to adapting old recipe migrations running on new installs to the new settings feature.

@jptissot and I are trying to work through what is the correct procedure to update recipe migrations  and will advise

**Comment by jptissot** [comment]
I have found the solution and issue that created this weirdness. Will post details in a bit.

**Comment by hishamco** [comment]
The **Create new type** button is fine, but I will check the badge for the stereotypes may it's UI related issue

**Comment by jptissot** [comment]
Turns out that the issue was caching. Here are the steps I took to successfully migrate my sites and recipes. This [commit](https://github.com/OrchardCMS/OrchardCore/commit/3501367a6dc7dbfd2fef8de36ab2df5862e842ba) was the one that caused some issues for me.

1. Setup the site and tenants while on a previous version of the code (in my case it was 1.0.0-beta3-72520). Run recipes that had the previous format for settings.
2. Stop site. run `dotnet clean`, `dotnet build` and `dotnet run` in order
3. Start site, migration runs to transfer the part / type settings to new format
4. Export recipes with the new format.


The issue I had initially was because I did not do a `dotnet clean` before running the site.
1- I had some tenants setup in my dev environment using an older format of the settings.
2- I ran dotnet run after having updated to the latest dependencies
3- The migration script migrated half way, putting my tenant in a weird state.
4- The only way to go forward was to wipe everything and restart since I was now in a bad state.

Hope this helps someone.
